pkg/leeway/scripts.go

```diff
@@ -210,7 +210,7 @@ func (p *Script) synthesizePackagesWorkdir(buildCtx *buildContext) (path string,
 		}
 
 		var out []byte
-		cmd := exec.Command("tar", getTarArgs("-xzf", br)...)
+		cmd := exec.Command("tar", "--sparse", "-xzf", br)
```

Comment:
should this only add `--sparse` on GOOS=linux?

Reply:
Fixed